{{cookiecutter.module_name}}/Makefile

```diff
@@ -100,18 +100,21 @@ json/format: | guard/program/jq
 	$(FIND_JSON) | $(XARGS) bash -c 'echo "$$(jq --indent 4 -S . "{}")" > "{}"'
 	@ echo "[$@]: Successfully formatted JSON files!"
 
-docs/%: README_PARTS := _docs/MAIN.md <(echo) <(terraform-docs markdown table .)
-docs/%: README_FILE ?= README.md
+tfdocs-awk/install: $(BIN_DIR)
+tfdocs-awk/install: ARCHIVE := https://github.com/plus3it/tfdocs-awk/archive/master.tar.gz
+tfdocs-awk/install:
+	$(CURL) $(ARCHIVE) | tar -C $(BIN_DIR) --strip-components=1 --wildcards '*.sh' --wildcards '*.awk' -xzvf -
+	chmod -R 755 $(BIN_DIR)/*
```

Comment:
Is this necessary? If the scripts are checked-in with the execute bit, they should be good to go...

Reply:
didn't realize setting the execute bit persisted across `git push/clone/etc.` updated